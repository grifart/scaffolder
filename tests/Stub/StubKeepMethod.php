<?php


namespace Grifart\ClassScaffolder\Test\Stub;


final class StubKeepMethod
{

	/**
	 * Some PHPDoc!
	 * @return mixed
	 */
	public function somethingToBeKept(): mixed {
		return 'whatever';
	}

}