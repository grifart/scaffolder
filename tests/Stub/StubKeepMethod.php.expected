namespace Grifart\ClassScaffolder\Test\Stub;

final class StubKeepMethod
{
	/**
	 * This method is kept while scaffolding.
	 */
	public function newMethod(): void
	{
		// Implement method here
	}


	/**
	 * Some PHPDoc!
	 * @return mixed
	 */
	public function somethingToBeKept(): ?mixed
	{
		return 'whatever';
	}
}